From 407f67102d955c205d8ce6ae3453d876e3b06f74 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 19 Apr 2020 23:03:44 +0300
Subject: [PATCH] [SW-0] make: include <device>_reg_map.h for all

The reg map files are provided and maintain common naming of the same
functonal registers across devices.
This enables to remove parts of the runtime code and resolve to the same
addresses in compile time. Helps reduce the binary size.

Change-Id: I907dcd063a5a0d2b7a7a2ccb7309e904b974df33
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index c7b3b6a..0f0ee70 100644
--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,7 @@
 SPECS_DEV := $(shell echo $(HL_DEV) | cut -d'_' -f1)
 CFLAGS += -I$(dir $(BR2_EXTERNAL))/specs/$(SPECS_DEV)/asic_reg
 CFLAGS += -I$(dir $(BR2_EXTERNAL))/embedded/specs
+CFLAGS += -include $(dir $(BR2_EXTERNAL))/embedded/specs/$(SPECS_DEV)/$(SPECS_DEV)_reg_map.h
 CFLAGS += -Wall -Werror
 CFLAGS += -g -c
 CFLAGS += -I ./include/ -I ./ -I ./common/ -I ./$(HL_DEV)/
